appservice/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "vscode-azureappservice",
     "author": "Microsoft Corporation",
-    "version": "0.4.0",
+    "version": "0.4.1",
```

Comment:
Adding `editScmType` feels more like a 'feature' than a 'bug fix' (in which case it should be '0.5.0')

but not a huge deal either way

Reply:
Okay, will we be including auto-detect deploying and Java/C# support to release 5.0.0 as well?

Reply:
Will we be including auto-detect deployment in version 5.0.0 as well?

Reply:
We can do whatever you want :)